app/depict/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2015. NextMove Software Ltd
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>cdk-app</artifactId>
+        <groupId>org.openscience.cdk</groupId>
+        <version>1.5.12-SNAPSHOT</version>
```

Comment:
Ah `${project.version}` (really `${project.parentVersion}`) actually comes from this tag so it's the only place you must specify it.